types/lodash/index.d.ts

```diff
@@ -12465,71 +12465,69 @@ declare namespace _ {
          * @param defaultValue The value returned if the resolved value is undefined.
          * @return Returns the resolved value.
          */
-        get<TResult>(
-            object: any,
-            path: Many<StringRepresentable>,
-            defaultValue: TResult
-        ): TResult;
+        // when object is empty - definitely default value will be returned
```

Comment:
Remove this comment (and other similar comments) - it was useful for explaining things in the github issue, but not very appropriate here.

Reply:
ok, they are duplicated in tests, but here they can confuse autocompleter